source/backend/api/Services/AcquisitionFileService.cs

```diff
@@ -154,6 +154,11 @@ public PimsAcquisitionFile UpdateProperties(PimsAcquisitionFile acquisitionFile)
             return _acqFileRepository.GetById(acquisitionFile.Id);
         }
 
+        private static void UpdateFileNumber(PimsAcquisitionFile acquisitionFile)
```

Comment:
This doesn't seem related to these changes, should this be here?

Reply:
just a warning about alpha-order maybe?

Reply:
or private static vs private perhaps.

Reply:
yep, just a warning fix on the static

Reply:
This change is no longer relevant so I would revert. Not sure how you have that since my PR (merged a few days ago) removed that method (because number generation was moved to repository). Maybe you need to rebase?

Reply:
good to know. I will check double check that is no longer necessary